cmd/destroy/destroy.go

```diff
@@ -94,6 +87,7 @@ type destroyCommand struct {
 	k8sClientProvider okteto.K8sClientProvider
 	configMapHandler  configMapHandler
 	oktetoClient      *okteto.OktetoClient
+	getDestroyer      func(*kubernetes.Clientset, okteto.K8sClientProvider, destroyer, executor.ManifestExecutor, *okteto.OktetoClient, *Options) (destroyInterface, error)
```

Comment:
I think we can infer `kubernetes.Clientset` from `okteto.K8sClientProvider`. This reduces the number of params which is quite large for a function